packages/create-llama/templates/components/workflows/typescript/deep_research/workflow.ts

```
@@ -114,10 +110,10 @@ Create a well-structured outline for the research report that covers all the ans
 type ResearchQuestion = { questionId: string; question: string };
 type ResearchResult = ResearchQuestion & { answer: string };
 
-class PlanResearchEvent extends WorkflowEvent<{}> {}
-class ResearchEvent extends WorkflowEvent<ResearchQuestion[]> {}
-class ReportEvent extends WorkflowEvent<{}> {}
-class StopEvent extends StopEventBase<AsyncGenerator<ChatResponseChunk>> {}
+// class PlanResearchEvent extends WorkflowEvent<{}> {}
+const planResearchEvent = workflowEvent<{}>();
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)